pkgs/by-name/li/libportal/package.nix

```diff
@@ -68,6 +69,15 @@ stdenv.mkDerivation rec {
     qt6Packages.qtbase
   ];
 
+  patches = lib.optionals (variant == "qt6") [
```

Comment:
Unless there would be too many rebuilds, might be better to make this unconditional so that we do not forget to remove this on update.